README.MD

```diff
@@ -81,7 +81,8 @@ Example:
 |direction|Language direction|string|true|
 |locale|Locale string|string|true|
 |portal|`Portal` created by ApplicationBase|[Portal](https://developers.arcgis.com/javascript//latest/api-reference/esri-portal-Portal.html)|true|
-|results|ApplicationBase query results|[ApplicationBaseResults](#applicationbaseresults)|true|
+|results|ApplicationBase query results|[ApplicationBaseConfig](#applicationbaseconfig)|true|
```

Comment:
These ones don't seem right. It should be `ApplicationBaseResults` for results and settings needs a new definition